EWS O365: Fixed missing the eml extension (#30179)
demisto/content: Packs/MicrosoftExchangeOnline/ReleaseNotes/1_2_29.md

#### Integrations

##### EWS O365

> - Fixed an issue where the attachment name was missing the eml extension.

Reviewer: ```suggestion
- Fixed an issue where the attachments were missing the file extension.
```
